src/librustdoc/markdown.rs

```diff
@@ -55,26 +54,22 @@ pub fn render(
         css.push_str(&s)
     }
 
-    let input_str = match load_string(&input, diag) {
+    let input_str = match load_string(input, diag) {
         Ok(s) => s,
-        Err(LoadStringError::ReadFail) => return 1,
-        Err(LoadStringError::BadUtf8) => return 2,
+        Err(LoadStringError::ReadFail) => return Err("Read error: read failed".to_owned()),
+        Err(LoadStringError::BadUtf8) => return Err("Read error: invalid UTF8".to_owned()),
```

Comment:
`load_string` already prints error messages so this results in duplicated messages. We can probably just use `std::fs::read_to_string` here.

Reply:
Good point, I'll check how to prevent it.